### ISSUE TITLE
Fails setting ZRAM dependency after migration to OpenHAB3 (not sure if I was using ZRAM in the base case)

### ISSUE THREAD

**Issue by BigGeorgeTx** [openhab/openhabian#1453]
## Issue information:
I tried to migrate to OpenHAB3 using openhabian.  It seems like the install was working, but then it failed.  
I can force OpenHAB3 to run with openhab-cli start --debug.  It runs and I can access the new UI.  But that only lasts for a while and then it stops.

As instructed, I increased the openhabian debug to maximum.  It seems to be having a problem adding zram dependency.  I don't recall if I ever had ZRAM set up on this PI or not.  This was my first venture into SBC and OpenHAB.  I remember being a little concerned about SD card wearout and reading about ZRAM, but honestly can't remember if I ever tried to set it up or not.  When I enter systemctl status zram-config.service, it says there is no such service.  I tried uninstalling and reinstalling zram using the openhabian-config menu, but encountered errors either way.  

![image](https://user-images.githubusercontent.com/13175983/106965629-7c238000-6709-11eb-92a1-de8802e4cb93.png)

Trying to uninstall:
`2021-02-04_13:24:00_MST [openHABian] Removing zram service... + cond_redirect zram-config stop
+ [[ -n '' ]]
+ echo -e '\n\033[90;01m$ zram-config stop \033[39;49;00m'

$ zram-config stop
+ zram-config stop
/opt/openhabian/functions/helpers.bash: line 31: zram-config: command not found
+ return 127
+ echo 'FAILED (stop zram)'
FAILED (stop zram)
+ return 1
+ '[' 1 -ne 0 ']'
+ whiptail --msgbox 'There was an error or interruption during the execution of:\n  "30 | System Settings"\n\nPlease try again. If the error persists, please read /opt/openhabian/docs/openhabian-DEBUG.md or https://github.com/openhab/openhabian/blob/main/docs/openhabian-DEBUG.md how to proceed.' 14 80
`
And trying to install
`+ echo '2021-02-04_13:07:38_MST [openHABian] Refusing to install zram as it is already installed, please uninstall and then try again... EXITING'
2021-02-04_13:07:38_MST [openHABian] Refusing to install zram as it is already installed, please uninstall and then try again... EXITING
+ return 1
+ '[' 1 -ne 0 ']'
+ whiptail --msgbox 'There was an error or interruption during the execution of:\n  "30 | System Settings"\n\nPlease try again. If the error persists, please read /opt/openhabian/docs/openhabian-DEBUG.md or https://github.com/openhab/openhabian/blob/main/docs/openhabian-DEBUG.md how to proceed.' 14 80
+ return 0
+ true
+ show_main_menu
+ local choice
+ local version
`

## Debug information:
Please provide all of the output provided by utilizing the `debugmode=maximum`
parameter in the `/etc/openhabian.conf` file. For more information on how to
accomplish this, please see [openhabian-DEBUG.md](https://github.com/openhab/openhabian/blob/main/docs/openhabian-DEBUG.md#create-a-debug-log).

`2021-02-04_12:59:15_MST [openHABian] Setting up openHAB service... + zram_dependency install openhab
+ local zramServiceConfig=/etc/systemd/system/zram-config.service
+ local install=yes
+ [[ -f /etc/ztab ]]
+ [[ install == \i\n\s\t\a\l\l ]]
+ shift 1
+ [[ openhab == \r\e\m\o\v\e ]]
+ for arg in "$@"
+ [[ yes == \y\e\s ]]
+ grep -qs openhab.service /etc/systemd/system/zram-config.service
++ timestamp
++ date +%F_%T_%Z
+ echo -n '2021-02-04_12:59:15_MST [openHABian] Adding openhab to zram service dependencies... '
2021-02-04_12:59:15_MST [openHABian] Adding openhab to zram service dependencies... + cond_redirect sed -i -e '/^Before/s/$/ openhab.service/' /etc/systemd/system/zram-config.service
+ [[ -n '' ]]
+ echo -e '\n\033[90;01m$ sed -i -e /^Before/s/$/ openhab.service/ /etc/systemd/system/zram-config.service \033[39;49;00m'

$ sed -i -e /^Before/s/$/ openhab.service/ /etc/systemd/system/zram-config.service
+ sed -i -e '/^Before/s/$/ openhab.service/' /etc/systemd/system/zram-config.service
sed: can't read /etc/systemd/system/zram-config.service: No such file or directory
+ return 2
+ echo 'FAILED (sed add dependency)'
FAILED (sed add dependency)
+ return 1
`

## System information:
HW you are running on (Raspberry Pi, amd64).
Running openhabian on a Pi4B

`cat /etc/os-release`
[13:07:45] openhabian@openhab:/etc$ cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

 Also include the output of
 and `uname -m`.
[13:11:44] openhabian@openhab:/etc$ uname -m
armv7l

**Comment by mstormi** [comment]
You're choking on a number of issues unrelated to openHABian here.
You must not start openHAB manually but use the existing service (`systemctl status/start/stop openhab`).
Then all your trouble is because ZRAM is not installed in the first place.
Use menu 38 with debugmode=maximum so you can see if it proceeds.

Please next time do not open a Github issue unless you're convinced there is a problem *in* openhabian. Github is not a support ticket system, make a post on the forum instead.

**Comment by BigGeorgeTx** [comment]
I apologize for putting this in the wrong form.  I misunderstood the discussion on 
![image](https://user-images.githubusercontent.com/13175983/107058414-35319b00-679a-11eb-80e0-dfe750285bcf.png) and thought that because I had run into a problem using openhabian, I should address it here rather than in the OpenHAB community.  I have noted the error I got trying to install ZRAM there and will look there for help in finding a solution.

Should I delete this issue that I raised, or it is better to just leave it alone at this point?
